deriva/core/ermrest_model.py

```diff
@@ -23,6 +24,12 @@ class NoChange (object):
 # singletone to use in APIs below
 nochange = NoChange()
 
+class UpdateMappings (str, Enum):
+    """Update Mappings flag enum"""
+    no_update = ''
+    deferred = 'deferred'
+    immediate = 'immediate'
+
```

Comment:
@karlcz think this is okay this way? or would it be better to have variable instances of the enum of say `no_update = UpdateMappings.no_update` analogous to your `nochange = NoChange()` instance?

Reply:
I think this is fine. You can use it as `UpdateMappings.no_update` etc.

Reply:
However, I'm not sure I like the empty string...  I think I'd also spell it out.

Reply:
Nevermind, I see you did it to make the "falsish" test easier to write below...